### DISCUSSION TITLE
Generate charts

### DISCUSSION THREAD

**Discussion by amigolj** [Laravel-Backpack/community-forum#1176] [General]
Hi, I gather data in DB and want to dynamically generate some charts in one page. I have read the docs but I see that for each chart I should create a separate controller.
 Is there perhaps another way?

**Comment by jcastroa87** [comment]
Hello @amigolj 

You can find information about how to achieve this [here](https://github.com/Laravel-Backpack/community-forum/issues/417)

Let me know if that's helpful for you.

Cheers.

**Comment by jcastroa87** [comment]
Due to not activity, I will close this issue, but please feel free to re-open or create a new one if needed.

Cheers.
